akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -441,6 +466,16 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
       .toMat(new AeronSink(outboundChannel(outboundContext.remoteAddress), ordinaryStreamId, aeron, taskRunner))(Keep.right)
   }
 
+  def outboundLarge(outboundContext: OutboundContext): Sink[Send, Future[Done]] = {
+    // TODO not sure about this, just cargo culting here
```

Comment:
it's right, they are the same as the ordinary messages aside from a few parameters of the AeronSink/AeronSource